### ISSUE TITLE
Unexpected token error with %i array symbol

### ISSUE THREAD

**Issue by gponsu** [bbatsov/rubocop#2187]
I think that rubocop is giving a false positive here:

```
app/models/concerns/json.rb:56:8: E: unexpected token tSTRING_CONTENT
    %i(name auth_token created_at updated_at)
       ^^^^^^^^^^^
```

The code:
```ruby
  def to_json
    %i(name auth_token created_at updated_at)
  end
```

**Comment by bbatsov** [comment]
Works fine for me. Post here the output from `rubocop -V`.

**Comment by gponsu** [comment]
Sorry, I'm an idiot, I was using version 1.9.3 of Ruby, which does not support %i :sweat_smile:
